test/morphing.jl

```diff
@@ -65,11 +65,9 @@ end
 
     function matrix(args...; do_transpose = false, do_action = :stroke)
         fontsize(50)
-        str =
-            L"$\begin{equation}\left[\begin{array}{ccc}\alpha & \beta & \gamma \\x^{2} & \sqrt{y} & \lambda \\1 & 2 & y \\\end{array}\right]\end{equation}$"
+        str = L"$\begin{equation}\left[\begin{array}{ccc}\alpha & \beta & \gamma \\x^{2} & \sqrt{y} & \lambda \\1 & 2 & y \\\end{array}\right]\end{equation}$"
```

Comment:
do you mind removing this file from the PR? Just to keep this PR cleaner

Reply:
This is something that I have to do or @TheCedarPrince will do that?
Because I dont know how to do this :sweat_smile: